### PR TITLE
added an avatar component

### PR DESCRIPTION
**Pull request by Lailax**
added fake wrapper

### CODE REVIEW

File: app/views/page/front.twig [hunk 1]
```diff
@@ -17,7 +18,25 @@
...
       {{ headline.archive() }}
     </div>
-  </section>
+
   {{ overview.default() }}
+   </section>
+
+  <section class="section">
+    <div class="section__content">
+      {{ headline.archive() }}
```
Comment (guddii) [inline app/views/page/front.twig]: Headline does not print the headline of this section

File: app/components/section/section.scss [hunk 2]
```diff
@@ -6,3 +6,12 @@
   max-width: 150rem;
   margin: auto;
 }
+
+
+.meta {
+  padding: 5rem 0;
```
Comment (guddii) [inline app/components/section/section.scss]: You can use http://csscomb.com/online for css formatting

File: app/components/avatar/avatar.scss [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  border-radius: 50em;
+  border-color: var(--grayDefault);
+  border-width: 1px;
+  border-style: solid;
+  display: inline-block;
+}
+
+
+.avatar__image{
+  border-radius: 50em;
```
Comment (guddii) [inline app/components/avatar/avatar.scss]: You can use http://csscomb.com/online for css formatting

File: app/components/avatar/avatar.scss [hunk 4]
```diff
@@ -0,0 +1,16 @@
+.avatar{
+  border-radius: 50em;
```
Comment (guddii) [inline app/components/avatar/avatar.scss]: You can use http://csscomb.com/online for css formatting

File: app/components/section/section.scss [hunk 5]
```diff
@@ -6,3 +6,12 @@
   max-width: 150rem;
   margin: auto;
 }
+
+
+.meta {
```
Comment (guddii) [inline app/components/section/section.scss]: This `.meta` is a component itself

File: app/views/page/front.twig (hunk 1 above)
Comment (nkleemann) [inline app/views/page/front.twig]: fixed, see: https://github.com/guddii/floidtv/commit/350381ef3515dd92b600b8e5480dfd35331c6ac2

**Pull request merged**